config/routes.rb

```diff
@@ -6,5 +6,13 @@
       resources :comments, :likes
     end
   end
-  # post '/users/:user_id/posts/:post_id/likes', to likes#create
+  namespace :api do
+    namespace :v1 do
+      resources :users do
+        resources :posts do
+          resources :comments
+        end
+      end
+    end
+  end
```

Comment:
- Excellent job is done 👍🏽 
<img width="935" alt="Screenshot 2023-04-14 at 17 11 48" src="https://user-images.githubusercontent.com/58692590/232100020-697b2dcc-6262-43f6-80ec-11cf3fe76932.png">